src/main/java/Cody.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+public class Cody {
+
+    private static void echo() {
+        Scanner in = new Scanner(System.in);
+        String input = in.nextLine();
+        while (!input.equals("bye")) {
+            System.out.println("____________________________________________________________\n"
+                    + " " + input + "\n"
+                    + "_____________________________________________________________");
+            input = in.nextLine();
+        }
+    }
```

Comment:
Readability: can define a constant for the horizontal line to improve the readability